android/src/main/java/io/radar/flutter/RadarFlutterPlugin.java

```diff
@@ -208,9 +198,6 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull final Result result)
                 case "setAnonymousTrackingEnabled":
                     setAnonymousTrackingEnabled(call, result);
                     break;
-                case "setAdIdEnabled":
```

Comment:
For breaking changes, we add entries to `MIGRATION.MD`.  This repo doesn't have one yet, but let's create one.  It should be formatted like this: https://github.com/radarlabs/radar-sdk-ios/blob/master/MIGRATION.md

Reply:
We can put all of the renamed functions in there as well, like that latest ios MIGRATION files does

Reply:
Yes, @tjulien